ts/runtime.ts

```diff
@@ -0,0 +1,823 @@
+/*******************************************************************************
```

Comment:
This file seems is very similar to runtime.js (which is good!) but do we need to copy paste between the two when there are updates? Is there some way to avoid needing to do that?

Reply:
I think it is a problem, too. 
We need to copy paste it, but it is a simple way.
I have some ideas to solve it. See below.(Second one is better, I think.)

1, Just use TypeScript's runtime. ```tsc``` command outputs JavaScript's one.
  * pros: runtime.js of quality is guaranteed. If ```tsc``` can not parse it, we can not get runtime.js.
  * cons: We always need ```tsc``` command.

2, Use a template file like Shakespeare.
 * pros: We do not need ```tsc``` command.
            The content of runtime.js can be controlled by haskell.
 * cons: Fay depends on Shakespeare.

3, Use other formats like bash-here-document or sed.
 * pros: We do not need ```tsc``` command.
            Fay does not depend on another package. 
 * cons: We depends on another program.